src/components/dashboard/PendingTxs/styles.module.css

```diff
@@ -27,24 +28,11 @@
   text-align: center;
 }
 
-.columnTemplate {
-  grid-template-columns: repeat(12, 1fr);
-  grid-template-areas: 'nonce type type type type type type info info info confirmations icon';
-}
-
-.columnWrap {
-  white-space: normal;
-  word-break: break-word;
-}
-
 @media (max-width: 599.95px) {
-  .confirmationsCount {
-    padding: 4px var(--space-1);
-  }
-
-  .columnTemplate {
-    grid-template-areas:
-      'nonce type type type type type type type type type confirmations icon'
-      'empty info info info info info info info info info info info';
+  .txInfo {
+    width: 100%;
+    order: 4;
```

Comment:
I think as long as the other elements don't have an explicit order any value larger than 0 would work here.